src/routes/DiaryList.js

```diff
@@ -1,19 +1,43 @@
 import React from 'react'
 import { connect } from 'react-redux'
 import { Link } from 'react-router-dom'
+import Button from 'material-ui/Button'
+import AddIcon from '@material-ui/icons/Add'
+import Grid from 'material-ui/Grid'
+
+import { fetchDiaries } from '../services/localStorage'
 
 class DiaryList extends React.Component {
+  componentDidMount() {
+    this.props.dispatch({
```

Comment:
We should create an action creator for `loadDiaries`

Reply:
makes sense